### DISCUSSION TITLE
Using remote atom via teleport

### DISCUSSION THREAD

**Discussion by eliabruni** [gravitational/teleport#8571] [Q&A] [answered]
Hello!

Do you guys think it is possible to use [remote atom](https://atom.io/packages/remote-atom) via teleport?

I am trying to follow [these instructions](https://gist.github.com/NTag/9d9be611e03098c282241652894bda7f), however I am not sure how to expose `52698:localhost:52698`. Any help would be appreciated!

**Comment by webvictim** [comment, marked as answer]
Teleport doesn't currently support remote SSH forwarding (the `-R` flag) so I'm afraid this isn't possible.

**Reply by eliabruni** [reply]
Pity, but thanks for the answer @webvictim !
